src/padplus-manager/padplus-manager.ts

```diff
@@ -1102,13 +1102,26 @@ export class PadplusManager extends EventEmitter {
     roomId: string,
     memberId: string,
   ) {
-    log.silly(PRE, `roomAddMember : ${util.inspect(roomId)};${memberId}`)
+    log.silly(PRE, `roomAddMember: ${roomId}, ${memberId}`)
     if (!this.padplusRoom) {
       throw new Error(`no padplus Room.`)
     }
     await this.padplusRoom.addMember(roomId, memberId)
   }
 
+  public async roomAvatar (roomId: string) {
+    log.silly(PRE, `roomAvatar : ${roomId}`)
+    if (!this.cacheManager) {
+      throw new Error(`no cache manager`)
+    }
+    const room = await this.cacheManager.getRoom(roomId)
```

Comment:
we should use `roomRawPayload` or something else to get the room data, instead of get it directly from cache. If the data is not loaded, but valid, there will be exception here.